test/e2e/servicedns.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
(no changes necessary) I was not suggesting we create duplicate tests in E2E that are covered in integration. It was only that we could have a common place to host the DNS tests that can be used by integration and E2E similar to the common crud tester. I think this is okay for now, but ideally we could have common test code that checks the DNS object updates regardless if we are testing a fake load balancer for integration or a real load balancer for E2E. Also, then when we consolidate integration and managed E2E in the future, we can just remove the code path for one of the implementations.